### DISCUSSION TITLE
Django static files not loading

### DISCUSSION THREAD

**Discussion by abhishekairan** [vercel/community#7454] [Help]
### Summary

Hello i have deployed a web app made via django and its static files (css and js) are not being loading/rendering on site

[Github repo]
https://github.com/abhishekairan/Inventory_Showcase

[Vercel Site]
https://vivah-collection-568j0q8y0-abhishekairans-projects.vercel.app/

### Example

https://vivah-collection-568j0q8y0-abhishekairans-projects.vercel.app/

### Steps to Reproduce

Clone the repo and deploy

**Comment by pawlean** [comment]
Hi @abhishekairan!

Can you check your build settings? Make sure that you are pointing to the correct path for deployment. https://vercel.com/docs/deployments/configure-a-build#root-directory

[a comment by an automated account is not shown]
